docker-mysql-base.yml

```diff
@@ -1,8 +1,9 @@
+# https://docs.docker.com/compose/extends/#adding-and-overriding-configuration
+
 version: "3.1"
```

Comment:
Seeing we introduce new files here it would be cool to bump to something more recent (see https://docs.docker.com/compose/compose-file/compose-versioning/#compatibility-matrix)

Travis CI seems to [run the following](https://travis-ci.org/github/twingly/ecco/jobs/737992958#L31-L49) at the moment:

```
docker version
Client:
 Version:           18.06.0-ce
 API version:       1.38
 Go version:        go1.10.3
 Git commit:        0ffa825
 Built:             Wed Jul 18 19:09:54 2018
 OS/Arch:           linux/amd64
 Experimental:      false
Server:
 Engine:
  Version:          18.06.0-ce
  API version:      1.38 (minimum version 1.12)
  Go version:       go1.10.3
  Git commit:       0ffa825
  Built:            Wed Jul 18 19:07:56 2018
  OS/Arch:          linux/amd64
  Experimental:     false
```

So I guess we could bump to 3.7 at least.

Reply:
Bumping it along the introduction to new files now: 04fdf5ab88ccbeebfb46add33f2f983e5764cd2e